README.md

```diff
@@ -13,18 +13,6 @@ For current progress on Dojo 2, contribution guidelines and other information, p
   - [Testing](#testing)
 - [Licensing information](#licensing-information)
 
-## Deploying to gh-pages
-
-Assuming that you are working on a clone of the repository on the branch that you would like to deploy (typically `master`)
-
-This assumes that npm & typings & dojo-cli have been installed for the example applications.
-
-```shell
-./scripts/gh-pages-publish.sh
-```
-
-The latest version(s) will now be available [on github pages](https://dojo.github.io/examples).
-
 ## Features
 
 There are currently six examples:
```

Comment:
six -> seven

Reply:
It's actually six, the cli-command example has been removed - I've removed that line.